appium/common/helper.py

```diff
@@ -38,6 +35,19 @@ def appium_bytes(value, encoding):
         return value  # Python 2
 
 
+def get_dict_const_attr_and_val(cls):
```

Comment:
[want] better name (if you have)

Reply:
how about `extract_items` or `extract_attributes` ?

Reply:
thanks, will rename to ```extract_const_attributes```.
(```isupper()``` works for const.)